RecoTracker/TkSeedGenerator/plugins/BuildFile.xml

```diff
@@ -1,3 +1,5 @@
+<use name="PhysicsTools/TensorFlow"/>
+<use name="SimDataFormats/TrackingAnalysis"/>
```

Comment:
do we want a simulation dependency in this package?

Reply:
removed (moved the MCtruthSeeder in Validation)